fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -1337,26 +1409,446 @@ class A320_Neo_FCU_Pressure extends A320_Neo_FCU_Component {
 
 class A320_Neo_FCU_SmallScreen extends NavSystemElement {
     init(root) {
-        if (this.pressure == null) {
-            this.pressure = new A320_Neo_FCU_Pressure(this.gps, "SmallScreen");
+        if (this.pressure1 == null) {
+            this.pressure1 = new A320_Neo_FCU_Pressure(this.gps, "LeftBaro");
+        }
+        if (this.pressure2 == null) {
+            this.pressure2 = new A320_Neo_FCU_Pressure(this.gps, "RightBaro");
         }
     }
     onEnter() {
     }
     onUpdate(_deltaTime) {
-        if (this.pressure != null) {
-            this.pressure.update(_deltaTime);
+        if (this.pressure1 != null) {
+            this.pressure1.update(_deltaTime);
+        }
+        if (this.pressure2 != null) {
+            this.pressure2.update(_deltaTime);
         }
     }
     onExit() {
     }
     onEvent(_event) {
     }
     reboot() {
-        if (this.pressure) {
-            this.pressure.reboot();
+        if (this.pressure1) {
+            this.pressure1.reboot();
+        }
+        if (this.pressure2) {
+            this.pressure2.reboot();
         }
     }
 }
 
+// Based on ftp://climate1.gsfc.nasa.gov/wiscombe/Solar_Rad/SunAngles/sunae.f
```

Comment:
Does the FCU have automatic brightness adjustment IRL?

Reply:
No, but the ambient lighting has a great effect on it's appearance, which is what we adjust with this :D.